webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path')
+const Dotenv = require('dotenv-webpack');
+const { VueLoaderPlugin } = require("vue-loader");
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/main.js',
+  output: {
+    path: path.resolve(__dirname, './dist'),
+    filename: 'build.js',
+  },
+  mode: 'none',
+  module: {
+    rules: [
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader',
+      },
+    ],
+  },
+  resolve: {
+    alias: {
+      vue$: 'vue/dist/vue.esm-bundler.js',
+    },
+    extensions: ['*', '.js', '.vue', '.json'],
+  },
+  plugins: [
+    new Dotenv(),  // Load environment variables from a .env file
```

Comment:
We may not need to load environment variables from a file (or, more likely, may want to do different things for local/dev/prod). Look like there are [configs for this](https://www.npmjs.com/package/dotenv-webpack#properties) (systemvars, defaults). We can leave this as-is for now but noting for when we get closer to deploy